tests/testthat/test-vignette.R

```diff
@@ -0,0 +1,26 @@
+context("vignette renders")
```

Comment:
We shouldn't need tests for vignettes. Just for the actual functions. r cmd check includes a check to make sure the vignettes can be built. So you can just delete this file.